docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -174,21 +174,21 @@ This metric is a gauge. It has a constant value `1`, and contains a label called
 
 This metric is a counter that shows you the number of failed peer connection attempts. In most cases you will want to use the `rate` function to understand how often reconnections fail as this may be indicative of an issue or instability in your network.
 
-## Monitor your metrics
+## Interact with your meta-monitoring metrics
```

Comment:
Can we say Work with, Explore or Analyze here?

Reply:
Sure - I think "work with" is probably best since "explore" specifically just connotes the Grafana Explore and "Analyze" similarly connotes just querying them as opposed to setting alerts on them (at least in my opinion).